homeassistant/components/shelly/sensor.py

```diff
@@ -1116,6 +1116,15 @@ class RestSensorDescription(RestEntityDescription, SensorEntityDescription):
         state_class=SensorStateClass.MEASUREMENT,
         available=lambda status: status is not None,
     ),
+    "voltmeter_value": RpcSensorDescription(
+        key="voltmeter",
+        sub_key="xvoltage",
+        name="Voltmeter value",
```

Comment:
Maybe someone has a better idea for a name for this sensor? I used `Voltege value` as for `xcounter` and `xpercent` sensors.